packages/salesforcedx-vscode-soql/src/messages/i18n.ja.ts

```diff
@@ -31,5 +31,5 @@ export const messages = {
   error_unknown_error:
     'ERROR: %s. Unknown error. Open an issue and provide the error message details: https://github.com/forcedotcom/soql-tooling/issues/new/choose.',
   error_data_view_save:
-    'ERROR: We can’t save the file to the query-results directory. Make sure you have write permissions for the directory.'
+    'ERROR: We can’t save the file to the specified directory. Make sure you have write permissions for the directory.'
```

Comment:
@ekapner We no longer know the directory in which they are trying to save.

Reply:
Yep, looks good.